clients/ui/frontend/src/app/pages/modelCatalog/components/ModelCatalogFilters.tsx

```diff
@@ -23,6 +36,25 @@ const ModelCatalogFilters: React.FC = () => {
   }
   return (
     <Stack hasGutter>
+      <StackItem>
+        <Stack hasGutter>
+          <StackItem>
+            <Switch
+              id="model-performance-view-toggle"
+              label="Model performance view"
+              isChecked={performanceViewEnabled}
+              onChange={(_event, checked) => setPerformanceViewEnabled(checked)}
+              data-testid="model-performance-view-toggle"
+            />
+          </StackItem>
+          <StackItem>
+            <Content component={ContentVariants.small}>
+              Enable performance filters, display model benchmark data, and exclude unvalidated
+              models.
+            </Content>
+          </StackItem>
+        </Stack>
+      </StackItem>
```

Comment:
The mocks have a card border around this, I think you need to use the Card component in here somewhere.

<img width="366" height="150" alt="Image" src="https://github.com/user-attachments/assets/589c88a6-a4e5-4031-8d39-a24f7774d1c4" />